test/test_eos_tabulated.cpp

```diff
@@ -257,6 +262,98 @@ SCENARIO("SpinerEOS depends on rho and sie", "[SpinerEOS],[DependsRhoSie]") {
     steelEOS.Finalize();      // cleans up device memory
   }
 }
+
+SCENARIO("SpinerEOS and EOSPAC Serialization",
+         "[SpinerEOS][DependsRhoT][DependsRhoSie][EOSPAC][Serialization]") {
+  GIVEN("Eoses initialized with matid") {
+    SpinerEOSDependsRhoT rhoT_orig = SpinerEOSDependsRhoT(eosName, steelID);
+    SpinerEOSDependsRhoSie rhoSie_orig = SpinerEOSDependsRhoSie(eosName, steelID);
+    EOS eospac_orig = EOSPAC(steelID);
```

Comment:
I think we talked about this, but can you add another EOSPAC class and see if it can serialize/de-serialize properly?

Reply:
Yeah we talked about this. Let me get that in in a subsequent MR after understanding the EOSPAC properties in a self-contained test.